File: packages/react-native-reanimated/android/src/main/java/com/swmansion/worklets/WorkletsMessageQueueThread.java
package com.swmansion.worklets;

import com.facebook.proguard.annotations.DoNotStrip;

@DoNotStrip
public class WorkletsMessageQueueThread extends WorkletsMessageQueueThreadBase {
  @Override
  public boolean runOnQueue(Runnable runnable) {
    return messageQueueThread.runOnQueue(runnable);
  }

  @Override
  public boolean isIdle() {
    return messageQueueThread.isIdle();
  }
}

File: packages/react-native-worklets/android/src/main/java/com/swmansion/worklets/WorkletsMessageQueueThread.kt
package com.swmansion.worklets

import com.facebook.proguard.annotations.DoNotStrip

@DoNotStrip
class WorkletsMessageQueueThread : WorkletsMessageQueueThreadBase() {
    override fun runOnQueue(runnable: Runnable): Boolean = messageQueueThread.runOnQueue(runnable)

    override fun isIdle(): Boolean = messageQueueThread.isIdle()
}
